lib/beaker/options/parser.rb

```diff
@@ -297,7 +297,7 @@ def parse_hosts_options
         hosts_file_content = begin
           bhg_cli = BeakerHostGenerator::CLI.new(host_generator_options)
           bhg_cli.execute
-        rescue BeakerHostGenerator::Exceptions::Error,
+        rescue BeakerHostGenerator::Exceptions::Error, # rubocop:disable Lint/ShadowedException
           BeakerHostGenerator::Exceptions::InvalidNodeSpecError => e
```

Comment:
Please fix the cop instead. It's complaining (rightfully so) that `InvalidNodeSpecError` is a subclass of `Error` and thus the rescue is redundant.
```suggestion
        rescue BeakerHostGenerator::Exceptions::Error => e
```

Reply:
done